### ISSUE TITLE
generate_python_bindings.sh dependencies 

### ISSUE THREAD

**Issue by danielk333** [Space-Systems/neptune#22]
The script in `OPI/tools/generate_python_bindings.sh` depends on `h2xml` and `xml2py`, however looking into these packages they seem to be semi-abandoned (on [ubuntu launchpad](https://launchpad.net/ubuntu/+source/python-ctypeslib) the latest release was 2015 and hence they dont show up in e.g. AUR on arch), trying to find the source code the only refence avalible is [this repository](https://github.com/trolldbois/ctypeslib) which seems to have switched use-case as it does not supply `h2xml` and `xml2py` as entry-points anymore but rather `clang2py`.

Any ideas? Or did i just miss what the actual package is? xD

**Comment by danielk333** [comment]
Ah i found the svn repository for [ctypeslib](https://svn.python.org/projects/ctypes/trunk/ctypeslib/#egg=ctypeslib-dev) (the repository i linked before is ctypeslib2)

**Comment by danielk333** [comment]
Cloning the svn code revealed that its only python 2 compatible so this is probably a no-go as a dependency

**Comment by ckebschull** [comment]
Hi Daniel, yes you are right that code has not been touched in a while. Maybe you could ask your question on the OPI side of things. Then my colleague @mmoeckel could respond. Please also note that we are currently stuck on OPI-2015  and OPI has been further developed.

**Comment by danielk333** [comment]
#1 should resove this issue right?

**Comment by ckebschull** [comment]
That is correct. I hope to soon be able to look at it, but this is more complex as NEPTUNE is used in other projects, which still run on OPI-2015.

**Comment by danielk333** [comment]
Ah i see. But that should not be a big problem since its under version control and those projects can just check out a particular release/hash/tag using OPI-2015?
